src/ProjectTemplates/Web.ProjectTemplates/content/Components-CSharp/wwwroot/appsettings.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Is it expected these files are in wwwroot? Are they sent to the client?

Reply:
Yes, these are the config files for when the app is running inside the browser in webassembly. It's a bit challenging with the unified project because you want settings from running on webassembly to be different from the ones on server, as you don't want to potentially pass secrets down if someone puts them on the server appsettings.

As I am working through the multitargeting support, we might end up with some conventions similar to the ones on Maui, and we could potentially do something like appSettings.server.json and appSettings.browser.json